### PR TITLE
New domains for parent repo

### PR DESCRIPTION
**Pull request by NewAlexandria**
All recent domain additions. Includes
* edge and pixel domains with Akamai, appspot, and others
* additional messenger domain
* additional fbsbx CDN domains

## Changes
* I've attempted to dedupe the domains, [very much like the outcome of this discussion today](https://github.com/mozilla/contain-facebook/pull/157#issuecomment-431138717).  This include whatsapp domains.
* I have also organized the domains so that they are in more maintainable groups.  This make _this PR's diff_ harder to read, but will make things better going forward.

Look for comments, inline, about the additions

### CODE REVIEW

**Comment by esaruoho** [comment]
hey @NewAlexandria looks like these two are closed and ignored&closed:

- #141 (VKontakte PR) has been closed and deleted, as VK has nothing to do with FB
- #147 (FB Tor/Onion PR) has been closed after some serious back'n'forth.

and this PR causes conflicts:

- #151 is causing conflicts with `src/background.js`

**Comment by NewAlexandria** [comment]
Updated the PR and resolved tests. @groovecoder @groovecoder-mozilla-owner @ryanfeeley 

Thanks @esaruoho

**Comment by esaruoho** [comment]
hey @NewAlexandria looks like you're having `src/background.js` conflicts, and now that PR157 has been merged in, you might wanna remove that from your commit..

**Comment by NewAlexandria** [comment]
I've updated it @esaruoho.   FYI @ShivangiKakkar and @groovecoder — thanks for your time while you're in here looking at PRs.

* I've attempted to dedupe the domains, [very much like the outcome of this discussion today](https://github.com/mozilla/contain-facebook/pull/157#issuecomment-431138717).  
* I have also organized the domains so that they are in more maintainable groups.  This make _this PR's diff_ harder to read, but will make things better going forward.

File: src/background.js [hunk 1]
```diff
@@ -3,11 +3,19 @@ const FACEBOOK_CONTAINER_NAME = "Facebook";
 const FACEBOOK_CONTAINER_COLOR = "blue";
 const FACEBOOK_CONTAINER_ICON = "briefcase";
 const FACEBOOK_DOMAINS = [
-  "facebook.com", "www.facebook.com", "fb.com", "fbcdn.net", "fbsbx.com",
-  "instagram.com", "www.instagram.com",
-  "messenger.com", "www.messenger.com",
-  "whatsapp.com", "www.whatsapp.com", "web.whatsapp.com", "cdn.whatsapp.net", "www-cdn.whatsapp.net",
+  "facebook.com", "www.facebook.com", "facebook.net", "fb.com", 
+  "fbcdn.net", "fbcdn.com", "fbsbx.com", "tfbnw.net",
+  "facebook-web-clients.appspot.com", "fbcdn-profile-a.akamaihd.net", "fbsbx.com.online-metrix.net", "connect.facebook.net.edgekey.net",
```
Comment (groovecoder) [inline src/background.js]: Do you have any example of urls to/for these domains?

**Comment by esaruoho** [comment]
@NewAlexandria could you update your PR description and code, since #157 and #151 have been landed, so maybe this PR is currently slightly outdated?

**Comment by groovecoder** [comment]
Good domain additions for links that may send users to other Facebook properties. And it includes the domains from #151 and #157 too. r+

**Comment by NewAlexandria** [comment]
Thanks

**Pull request merged**